components/PlantCard/index.js

```diff
@@ -0,0 +1,102 @@
+import { createGlobalStyle } from "styled-components";
+//components/List.js
+import styled from "styled-components";
+
+import { useState } from "react";
+import Image from "next/image";
+
+export default function PlantCard() {
+  const [plants, setPlants] = useState([
+    {
+      name: "Snake Plant",
+      image:
+        "https://images.pexels.com/photos/2123482/pexels-photo-2123482.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Monstera",
+      image:
+        "https://images.pexels.com/photos/6597437/pexels-photo-6597437.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Pothos",
+      image:
+        "https://images.pexels.com/photos/7663986/pexels-photo-7663986.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Spider Plant",
+      image:
+        "https://images.pexels.com/photos/4603327/pexels-photo-4603327.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Aloe Vera ",
+      image:
+        "https://images.pexels.com/photos/1671650/pexels-photo-1671650.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "ZZ Plant",
+      image:
+        "https://images.pexels.com/photos/7513150/pexels-photo-7513150.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Jade Plant",
+      image:
+        "https://images.pexels.com/photos/6045046/pexels-photo-6045046.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Hoya",
+      image:
+        "https://images.pexels.com/photos/14958768/pexels-photo-14958768.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Rubber Plant ",
+      image:
+        "https://images.pexels.com/photos/993018/pexels-photo-993018.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    },
+    {
+      name: "Philodendron",
+      image:
+        "https://images.pexels.com/photos/7663987/pexels-photo-7663987.jpeg",
+    },
+  ]);
+
```

Comment:
To have your data-object as initial value in the useState is correct, but it doesn't look very nice. I would suggest to store your plants in a const and create a separate file for the data. Then. pass the const to the useState.